src/utils.ts

```diff
@@ -129,13 +129,18 @@ export function getSubscriberStatusCssClassName(status: string) {
   return subscriberCssClass;
 }
 
+function replaceAddSigns(url: string): string {
+  return url.replace('+', '%2B');
+}
+
 export function extractParameter(
   location: any,
   parseQueryStringFunction: any,
   paramLowercase: string,
   paramUppercase?: string,
 ) {
-  const parsedQuery = location && location.search && parseQueryStringFunction(location.search);
+  const parsedQuery =
+    location && location.search && parseQueryStringFunction(replaceAddSigns(location.search));
```

Comment:
How about this?
```suggestion
    location && location.search && parseQueryStringFunction(encodeURIComponent(location.search));
```

Reply:
we talk with Andres and change the functionality